src/EnergyPlus/ResultsFramework.cc

```diff
@@ -854,10 +869,12 @@ namespace ResultsFramework {
 
         for (auto &item : outputs) {
             std::string datetime = item.first;
-            if (smallestReportingFrequency < OutputProcessor::ReportingFrequency::Monthly) {
-                datetime = datetime.replace(datetime.find(' '), 1, "  ");
-            } else {
-                convertToMonth(state, datetime);
+            if (rewriteTimestamp) {
+                if (smallestReportingFrequency < OutputProcessor::ReportingFrequency::Monthly) {
+                    datetime = datetime.replace(datetime.find(' '), 1, "  ");
+                } else {
+                    convertToMonth(datetime);
+                }
```

Comment:
Matching the previous CSV workflow requires some rewriting of the timestamp. This is only done if the ISO 8601 format is not requested.